### PR TITLE
New home page

### PR DESCRIPTION
**Pull request by GregoryConrad**

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -1,65 +1,96 @@
...
 import 'package:wpi_campus/model/repositories/firebase/firebase_data_repository.dart';
-import 'package:wpi_campus/ui/user_events.dart';
+import 'package:wpi_campus/ui/home_event_card.dart';
+import 'package:wpi_campus/ui/user_event_card.dart';
 
 final get = GetIt.instance;
 
+// todo consider using a material swatch instead of just primary color
+final _theme = ThemeData(primaryColor: Color.fromRGBO(172, 43, 55, 1));
+
```
Comment (GabrielDeml) [inline lib/main.dart:16]: Should we make a theme.dart somewhere?

File: lib/main.dart [hunk 2]
```diff
@@ -1,65 +1,96 @@
...
 void main() {
   get.registerSingleton<DataRepository>(FirebaseDataRepository());
   runApp(MaterialApp(
     title: 'WPI Event Manager',
-    theme: ThemeData(primaryColor: Color.fromRGBO(172, 43, 55, 1)),
-    home: MainPage(),
+    theme: _theme.copyWith(brightness: Brightness.light),
+    darkTheme: _theme.copyWith(brightness: Brightness.dark),
+    themeMode: ThemeMode.system,
+    home: _MainPage(),
```
Comment (GabrielDeml) [inline lib/main.dart:24]: That is a slick way of doing it. 

Would it be better having this in a settings page though?

Comment (GregoryConrad) [reply lib/main.dart:24]: This should adopt the theme to be dark mode/light mode depending on the system, but if you want to do this in a settings page, you could also do that too

File: lib/main.dart (hunk 1 above)
Comment (GregoryConrad) [reply lib/main.dart:16]: IMO for a simple one-liner like this that ties into the app widget, this is fine

Comment (GabrielDeml) [reply lib/main.dart:16]: Yee that makes sense. I think we should still have a theme.dart eventually when we get more complicated themes

**Review by GregoryConrad — COMMENTED** [review]
Looked it over and it appears to be fine. @GabrielDeml if you don't have any suggestions I think it is ready to merge

**Pull request merged**